sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/administration/models/DocumentModelBuildMode.java

```diff
@@ -12,6 +12,12 @@
 @Immutable
 public final class DocumentModelBuildMode extends ExpandableStringEnum<DocumentModelBuildMode> {
 
+    /**
+     * Creates a DocumentModelBuildMode object.
+     */
+    public DocumentModelBuildMode() {
```

Comment:
Since this is an `ExpandableStringEnum` this will need to be marked as deprecated and tell users to use the factory method.